griptape/drivers/vector/pgvector_vector_store_driver.py

```diff
@@ -0,0 +1,159 @@
+import uuid
+from typing import Optional
+from attr import define, field, Factory
+from griptape.drivers import BaseVectorStoreDriver
+from sqlalchemy.engine import Engine
+from sqlalchemy import create_engine, Column, String, JSON
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy.dialects.postgresql import UUID
+from sqlalchemy.orm import Session
+from pgvector.sqlalchemy import Vector
+
+
+@define
+class PgVectorVectorStoreDriver(BaseVectorStoreDriver):
+    """
+    A vector store driver to Postgres using the PGVector extension.
```

Comment:
Put the docstring description on the first line, I think it impacts how reference docs are generated. Same for functions throughout the class. See https://google.github.io/styleguide/pyguide.html#384-classes.